README.md

```diff
@@ -0,0 +1,197 @@
+# pysdccc
+
+This python packages provides a convenient way to execute the [sdccc test suite](https://github.com/Draegerwerk/sdccc/).
```

Comment:
please write SDCcc instead of sdccc, fix all occurrences.

Reply:
done

Reply:
There are still some occurrences of sdccc that could be changed to SDCcc